a1/src/main/java/ro/utcn/is/a1/controller/ConsoleController.java

```diff
@@ -0,0 +1,324 @@
+
+    @Override
+    public void run(String... args) throws Exception {
+        print("Welcome to the attendance tracking system.");
+        boolean done = false;
+        print("LOG IN :");
+        print("Username :");
+        String usernameInput = scanner.next().trim();
+        print("Password :");
+        String passwordInput = scanner.next().trim();
+
+           if(checkInputData(usernameInput, passwordInput)) {
+               while (!done) {
+                   print("Enter a command: ");
+                   print("      [users]");
+                   print("      [questions]");
+                   print("      [tags]");
+                   String command = scanner.next().trim();
+                   try {
+                       done = pickCommandList(command, userManagementService.getUser(usernameInput));
+                   } catch (UserNotFoundException userNotFoundException) {
+                       print("The user with the given ID was not found!");
+                   }
+               }
+           }
+           else{
+               print("WRONG PASSWORD!");
+           }
+        }
+
+    private boolean checkInputData(String usernameInput, String passwordInput){
+
+       User tempUser = userManagementService.getUser(usernameInput);
+       if(tempUser.getUsername().equals(usernameInput)){
+           if(tempUser.getPassword().equals(passwordInput)){
+               return true;
+           }
+           else{
+               return false;
+           }
+       }
+       else{
+           return false;
+       }
+    }
+
+
+private boolean pickCommandList(String command, User user) throws Exception{
+
+    switch (command) {
+        case "users":
+            print("Enter a command:");
+            print("     [list]");
+            print("     [add]");
+            print("     [update]");
+            print("     [remove]");
+            String command1 = scanner.next().trim();
+            handleCommandU(command1);
+            return false;
+        case "questions":
+            print("Enter a command:");
+            print("     [list]");
+            print("     [add]");
+            print("     [update]");
+            print("     [remove]");
+            print("     [search]");
+            String command2 = scanner.next().trim();
+            handleCommandQ(command2, user);
+            return false;
+        case "tags":
+            print("Enter a command:");
+            print("     [list]");
+            print("     [add]");
+            print("     [update]");
+            print("     [remove]");
+            print("     [filter questions]");
+            String command3 = scanner.next().trim();
+            handleCommandT(command3);
+            return false;
+        default:
+            print("Unknown command. Try again.");
+
+
+            return false;
+    }
+}
+    //-------------- HANDLE COMMANDS
+    private boolean handleCommandU(String command) {
```

Comment:
Naming convention could be better. Would have been better to call it "handleUserCommands", "handleListUsers", etc. The intent of the method should be very clear from the name.